rke/k8s_rke_system_images.go

```diff
@@ -5866,6 +5953,93 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
+		"v1.20.12-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.20.12-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
+			NginxProxy:                "rancher/rke-tools:v0.1.78",
+			CertDownloader:            "rancher/rke-tools:v0.1.78",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.78",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.15.10",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.15.10",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.15.10",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.1",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.13.0",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.17.2",
+			CalicoCNI:                 "rancher/mirrored-calico-cni:v3.17.2",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.17.2",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.17.2",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.17.2",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.17.2",
+			CanalCNI:                  "rancher/mirrored-calico-cni:v3.17.2",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.17.2",
+			CanalFlannel:              "rancher/coreos-flannel:v0.13.0-rancher1",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.17.2",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.1.1.0.1ae238a",
+			AciHostContainer:          "noiro/aci-containers-host:5.1.1.0.1ae238a",
+			AciOpflexContainer:        "noiro/opflex:5.1.1.0.1ae238a",
+			AciMcastContainer:         "noiro/opflex:5.1.1.0.1ae238a",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.1.1.0.1ae238a",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.1.1.0.1ae238a",
+			AciGbpServerContainer:     "noiro/gbp-server:5.1.1.0.1ae238a",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.1.1.0.1ae238a",
+			PodInfraContainer:         "rancher/mirrored-pause:3.2",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-0.43.0-rancher3",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.4.1",
+			CoreDNS:                   "rancher/mirrored-coredns-coredns:1.8.0",
+			CoreDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.1",
+			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
+			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
+		},
+		// Enabled in v2.6.2
```

Comment:
Since we're doing an out of band release for October patch versions after v2.6.2 is going to be released (hopefully today/tomorrow) - this comment should say `Enabled out of band post v2.6.2`.

Reply:
Same logic for remaining 3 comments as well. Rest looks good!

Reply:
Thanks, look good. I just noticed the versions coming from 2.5 https://github.com/rancher/kontainer-driver-metadata/pull/705 also need an updated comment. Reviewing there so that PR + this both get updated.

Reply:
Thank you for pointing that out. I updated the comments.